spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/Trade.java

```diff
@@ -62,7 +62,7 @@
 
 	private List<Instant> executionTimes;
 
-	@Interleaved
+	@Interleaved(lazy = true)
```

Comment:
can we keep one of them eager?

Reply:
I wanted to keep them both lazy because this would be 2-levels deep of lazy and I wanted to specifically touch that :/

Reply:
do we have any integration tests for eager interleaved children?

Reply:
At the moment all 3 of the generations of these tables are being linked together by these 2 levels of lazy links. 

I thought that it would be very unlikely that lazy works but eager doesn't considering how it is implemented (and then that specific implementation of lazy basically using all the same calls and functions as eager is verified by several unit tests). 

What do you think? I could of course add another table in here to be linked in eagerly, but I didn't want to create more clutter (because there is already Trade -> subtrade -> subtradecomponent and it may get messy).

Reply:
I think we need to have a test for eager read as well. Maybe a separate test class for that?

Reply:
fixed. ptal